frontend/src/e2e-test/specs/0_citizen/citizen-daycare-application-create.spec.ts

```diff
@@ -229,6 +230,33 @@ test('A validation warning is shown if preferred start date is not valid', async
   await citizenApplicationEditor.assertPreferredStartDateProcessingWarningIsShown(
     true
   )
+
+  await citizenApplicationEditor.setPreferredStartDate(
+    parse(
+      fullDaycareForm.form.serviceNeed.preferredStartDate,
+      'dd.MM.yyyy',
+      new Date()
+    )
+  )
+
+  await citizenApplicationEditor.assertPreferredStartDateInputInfo(false)
+})
+
+test('A validation error message is shown if preferred start date is not valid', async (t) => {
+  await t.useRole(enduserRole)
+  await t.click(citizenHomePage.nav.applications)
+  await citizenApplicationsPage.createApplication(
+    fixtures.enduserChildFixtureJari.id
+  )
+  await citizenNewApplicationPage.createApplication('DAYCARE')
+  applicationId = await citizenApplicationEditor.getApplicationId()
+
+  await citizenApplicationEditor.fillData(fullDaycareForm.form)
```

Comment:
Filling the full form takes quite a long time, could we just fill the required fields?